client/src/test/java/com/tencent/rss/client/impl/ShuffleReadClientImplTest.java

```diff
@@ -430,6 +430,32 @@ public void readTest14() throws Exception {
     readClient.close();
   }
 
+  @Test
+  public void readTest15() throws Exception {
```

Comment:
The test is to filter unexpected taskId, it's not this PR's target

Reply:
Update: write 6 blocks, read 3 of these blocks.